common/key.c

```diff
@@ -31,6 +32,7 @@ static int	v_event_grow(SCR *, int);
 static int	v_key_cmp(const void *, const void *);
 static void	v_keyval(SCR *, int, scr_keyval_t);
 static void	v_sync(SCR *, int);
+static char*	lookup_char_notation(char ch);
```

Comment:
Return `const char *`. Type of `ch` should be `int`.

Reply:
Fixed.